app/controllers/application_submissions_controller.rb

```diff
@@ -102,6 +102,12 @@ def show
     end
   end
 
+  def rejected
+    @record.reviewed = false
+    @record.move_to_dashboard
```

Comment:
This method happens to work but that's only a side effect here. I'd like this to be a learning experience instead of me just telling you what to do instead so I'll start by asking:
What does `move_to_dashboard` do for you here?

Reply:
It redirects you to the dashboard so that you can see the application in the not reviewed section.

Reply:
I don't think that's happening there. We'll chat on Slack.

Reply:
So, `record.reviewed = false` sets the record to unreviewed but doesn't save it in that state. You would need to call `save` before that property you set actually gets persisted. However, calling `record.move_to_dashboard` happens to have an `update_attributes` inside it, which saves the record.
You didn't need `move_to_dashboard` at all, you needed to just call `record.save`. Or, you can change the method `move_to_dashboard` so that it does this as well.

Reply:
This doesn't look resolved to me. Maybe you and @Anbranin decided it was resolved outside of github?

Reply:
yeah we fixed it.